libs/feedback/src/lib/feedback-rating/feedback-rating.component.ts

```diff
@@ -45,6 +50,7 @@ export class FeedbackRatingComponent implements OnInit, AfterViewInit {
   ngAfterViewInit() {
     setTimeout(() => {
       this.ratingsClass = 'ratings slideup';
+      this.ref.markForCheck();
```

Comment:
It is an elegant animations? Maybe use @angular/animations or pure css of that?

Also, should use [ngClass] instead of [class] here:
https://github.com/nycJSorg/angular-presentation/blob/72eb9a1e2f069c73c672df2815b67a77f37ae9a6/libs/feedback/src/lib/feedback-rating/feedback-rating.component.html#L9

Reply:
marked as TODO for now